Content.Client/_DV/CosmicCult/UI/Monument/MonumentMenu.xaml.cs

```diff
@@ -0,0 +1,190 @@
+using System.Linq;
+using System.Numerics;
+using Content.Client.Stylesheets;
+using Content.Client.UserInterface.Controls;
+using Content.Shared._DV.CCVars;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Prototypes;
+using Content.Shared._DV.CosmicCult;
+using Robust.Client.AutoGenerated;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Robust.Client.Player;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client._DV.CosmicCult.UI.Monument;
+[GenerateTypedNameReferences]
+
+public sealed partial class MonumentMenu : FancyWindow
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly IEntityManager _ent = default!;
+    [Dependency] private readonly IPlayerManager _player = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    private readonly SpriteSystem _sprite;
+
+    // All glyph prototypes
+    private readonly List<GlyphPrototype> _glyphPrototypes;
+    // All influence prototypes
+    private readonly IEnumerable<InfluencePrototype> _influencePrototypes;
+    private readonly ButtonGroup _glyphButtonGroup;
+    private ProtoId<GlyphPrototype> _selectedGlyphProtoId = string.Empty;
+    private HashSet<ProtoId<GlyphPrototype>> _unlockedGlyphProtoIds = [];
+    public Action<ProtoId<GlyphPrototype>>? OnSelectGlyphButtonPressed;
+    public Action? OnRemoveGlyphButtonPressed;
+
+    public Action<ProtoId<InfluencePrototype>>? OnGainButtonPressed;
+    private int _ccEntropyValue = 0;
```

Comment:
Comment or rename to make more obvious what this represents

```suggestion
    private int _entropyPerCultist = 0;
```